src/test/resources/test.csv

```diff
@@ -1,2 +1,2 @@
 "ADDRESS","AMOUNT","DELIVERYTIME"
-"Address{uuid=null, name='a', companyName='a', street='a', city='a', postCode='a', country='a'}","5","2021-12-01T19:01:55.080100+01:00"
+"Address{uuid='68c9791a-280a-4da0-b403-48b8d15f1301', name='a', companyName='a', street='a', city='a', postCode='a', country='a'}","5","2021-11-20T14:20:53.128120+01:00"
```

Comment:
You could use [Faker](https://www.baeldung.com/java-faker) to generate random data.

Reply:
might as well use that, will look into it